main.go

```diff
@@ -41,19 +41,18 @@ func main() {
 }
 
 func enableDND() {
-	fmt.Printf("Turning on \"Do not disturb\" mode for %d minutes \n", min)
+	fmt.Printf("Turning on 'Do not disturb' mode for %d minutes\n", min)
 
 	// we don´t do error checking
 	_, _ = exec.Command("sh", "-c", "defaults -currentHost write ~/Library/Preferences/ByHost/com.apple.notificationcenterui doNotDisturb -boolean true").Output()
-	_, _ = exec.Command("sh", "-c", "defaults -currentHost write ~/Library/Preferences/ByHost/com.apple.notificationcenterui doNotDisturbDate -date \"`date -u +\"%Y-%m-%d %H:%M:%S +000\"`\"").Output()
+	_, _ = exec.Command("sh", "-c", "defaults -currentHost write ~/Library/Preferences/ByHost/com.apple.notificationcenterui doNotDisturbDate -date '`date -u +'%Y-%m-%d %H:%M:%S +000'`'").Output()
```

Comment:
Actually, this quoting doesn´t work and breaks the program. Rewriting the code and including error handling now.

No worries.